src/Symfony/Component/Security/Core/Encoder/Argon2iPasswordEncoder.php

```diff
@@ -17,9 +17,30 @@
  * Argon2iPasswordEncoder uses the Argon2i hashing algorithm.
  *
  * @author Zan Baldwin <hello@zanbaldwin.com>
+ * @author Dominik Müller <dominik.mueller@jkweb.ch>
  */
 class Argon2iPasswordEncoder extends BasePasswordEncoder implements SelfSaltingEncoderInterface
 {
+    private $config;
+
+    /**
+     * Argon2iPasswordEncoder constructor.
+     *
+     * @param int $memoryCost
```

Comment:
since this is for PHP 7.1, you can move the types to the constructor's signature, and remove the docblock altogether.